src/Microsoft.Health.Dicom.Core/Messages/CustomTag/GetCustomTagResponse.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+
+namespace Microsoft.Health.Dicom.Core.Messages.CustomTag
+{
+    public class GetCustomTagResponse
+    {
+        public GetCustomTagResponse(CustomTagEntry customTagEntry)
+        {
+            CustomTag = customTagEntry;
+        }
+
+        public GetCustomTagResponse(IEnumerable<CustomTagEntry> customTagEntries)
+        {
+            CustomTags = customTagEntries;
+        }
+
+        /// <summary>
+        /// Retrieved custom tag when request specifies exact custom tag path.
+        /// </summary>
+        public CustomTagEntry CustomTag { get; }
```

Comment:
I'm curious why we need 2 properties? 
Could we also utilize CustomTags for single custom tag retrieving?

Reply:
Serializing an IEnumarable returns a list of json entries which isn't accurate per what a user requested when asking for a single tag.

Reply:
I would say we should probably just have different response objects for this case instead of overloading a single one for two cases.